oneanddone/tasks/migrations/0022_make_taskname_unique.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+from south.v2 import DataMigration
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        changed = set()
+        for task in orm.Task.objects.all():
+            value = getattr(task, 'name')
```

Comment:
@bobsilverberg .actually I found a similar piece of code which i modified.it used getattr and settr I used it without thinking